### DISCUSSION TITLE
Question about WCAG conformance - how accessible is the product / features

### DISCUSSION THREAD

**Discussion by BogdanCerovac** [ClaperCo/Claper#77] [General]
Hi,

I'm wondering if there is any information about conformance to Web Content Accessibility Guidelines?

The reason is that we are looking for accessible solution for people to participate in hybrid accessible events in the EU.

The documentation is missing any info about WCAG: https://docs.claper.co/search.html?q=WCAG and so is the code (https://github.com/search?q=repo%3AClaperCo%2FClaper%20WCAG&type=code), so I hope somebody can shed some light on this.

Thank you in advance and kind regards,

Bogdan

**Comment by alxlion** [comment]
Hello,

Indeed, there isn't documentation about WCA and the product is not yet optimized for accessibility.

**Reply by BogdanCerovac** [reply]
Thank you.

Does "Not yet" mean that accessibility is being planned for and maybe there is a timeline for it?

Kind regards, Bogdan

**Reply by alxlion** [reply]
It's not planned because it's not on the priority list. But I'll accept any contribution related to it.
